client/my-sites/menus/main.jsx

```diff
@@ -104,28 +104,37 @@ var Menus = React.createClass( {
 				)
 			},
 			associationChanged: {
-				menu: function( menu, location ) {
+				menu: function( mainMenu, mainLocation ) {
```

Comment:
Maybe we could remove the arguments so we can use directly `menu` and `location`?